src/index.html

```diff
@@ -12,12 +12,20 @@
 
         </div>
         <ul class="nav-items">
-            <li id="series-count">Series(0)</li>
-            <li>Latest</li>
-            <li>Featured</li>
+            <li class="series" id="item-count">Series(0)</li>
+            <li class="latest">Latest</li>
+            <li class="">Featured</li>
         </ul>
     </nav>
-
+    <main>
+      <ul class="select">
+        <li class="latest">Latest</li>
+        <li class="series">Series</li>
+      </ul>
```

Comment:
Wonderful work you have done 👏. I like the presentation of your work 👍. However, I noticed that you already have **Series (0)** **Latest** and **Featured** on your navbar and they already perform the functions that the `select ul` items are performing. Therefore, I suggest you consider removing this block from your html.